tasks/src/main/scala/dagr/tasks/picard/CollectHsMetrics.scala

```diff
@@ -57,6 +58,7 @@ class CollectHsMetrics(override val in: PathToBam,
     buffer.append("BAIT_SET_NAME=" + baitSetName.getOrElse(CollectHsMetrics.baitSetName(targets)))
     buffer.append("LEVEL=ALL_READS")
     buffer.append("PER_TARGET_COVERAGE=" + metricsFile(extension=CollectHsMetrics.PerTargetExtension, kind=PicardOutput.Text))
+    buffer.append("PER_BASE_COVERAGE=" + metricsFile(extension=CollectHsMetrics.PerBaseExtension, kind=PicardOutput.Text))
```

Comment:
I don't think we want this on by default, since it can be expensive to compute, and the output file can be large.  How about making a boolean constructor parameter to turn this on (it's off by default)?

Reply:
Sure. Makes sense. I updated with boolean that is off by default